src/main.ml

```diff
@@ -162,6 +162,18 @@ let tar_filename_prefix =
   let doc = "Filename prefix for tar/sha disk blocks" in
   Arg.(value & opt (some string) None & info ["tar-filename-prefix"] ~doc)
 
+let ssl_legacy =
+  let doc = "For TLS, allow all protocol versions instead of just TLSv1.2" in
+  Arg.(value & flag & info ["ssl-legacy"] ~doc)
+
+let good_ciphersuites =
+  let doc = "SSL good cipher suites for TLS protocols" in
+  Arg.(value & opt (some string) None & info ["good-ciphersuites"] ~doc)
+
+let legacy_ciphersuites =
+  let doc = "SSL legacy cipher suites for TLS protocols" in
```

Comment:
Let's not mention SSL here, but have the text the same as in sparse_dd:
`let doc = "Additional TLS ciphersuites allowed only if ssl-legacy is set" in`